### DISCUSSION TITLE
Multiline message without editor

### DISCUSSION THREAD

**Discussion by avidseeker** [profanity-im/profanity#1988] [Q&A] [answered]
I'm used to having line breaks in messages I send. Is it possible to have newlines within Profanity readline?

If not, is it possible to have profanity auto-opens editor (vim) every time I compose a message? I think this can work by sending the content of the `compose.md` file created by profanity every time it is updated and clearing it on successful send. This way Vim can be kept open on a separate window and have `:w` for sending the buffer.

**Comment by jubalh** [comment, marked as answer]
You press `ctrl+c` to open the editor. Or run `/editor`. Both are mentioned in the man pages.

Re multiline in readline see: https://github.com/profanity-im/profanity/issues/182 which can be found when searching `multiline` in the existing issues.
